src/config.js

```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import debug from 'debug';
+
+config();
+
+const debugConfig = debug('config');
```

Comment:
It's better to get used to do not directly use any primitive, like strings and numbers, except (zeros, empty strings, and one/minus one in some cases). Let's always use constants for primitives and try to find a meaningful name (e.g. `DEBUG_PREFIX_CONFIG`).

BTW, `debug` conventions are that you use your app name as the top-level debug prefix. E.g. we can use the package name. So I expect two constants here 😉